Makefile.defs

```diff
@@ -803,6 +801,13 @@ OPENSSH_DIR=$(CURDIR)/$(OPENSSH_BUILD_DIR)/openssh-prefix/src/openssh-build
 FBINK_BUILD_DIR=$(THIRDPARTY_DIR)/fbink/build/$(MACHINE)
 FBINK_DIR=$(CURDIR)/$(FBINK_BUILD_DIR)/fbink-prefix/src/fbink-build
 
+CURL_BUILD_DIR=$(THIRDPARTY_DIR)/curl/build/$(MACHINE)
+CURL_DIR=$(CURDIR)/$(CURL_BUILD_DIR)/curl-prefix/src/curl-build
+CURL_LIB=$(OUTPUT_DIR)/libs/libcurl.so.4
```

Comment:
While my Mac OS VM kind of broke and I deleted it, you still ought to write something like this. ;-)
```
SQLITE_LIB_EXT=$(if $(WIN32),.dll,$(if $(DARWIN),.dylib,.so))
SQLITE_LIB=$(OUTPUT_DIR)/libs/libsqlite3$(SQLITE_LIB_EXT)
```

Reply:
I took a shortcut because it's only (cross) built for Linux targets ;).

Reply:
cURL is the kind of thing that feels like it could be more generally useful. /shrug

Reply:
Anyway, just a thought. No objections. ;-)

Reply:
True ;).  7712b66